test/src/test/java/org/corfudb/infrastructure/log/StreamLogFilesTest.java

```diff
@@ -54,11 +53,10 @@ private ServerContext getContext() {
             .build();
     }
 
-    @Test
+    //@Test
```

Comment:
Should we remove or rewrite these tests?

Reply:
Yes, cleaning it up right now. Will add more relevant tests.